addons/mail/static/tests/chat_window/chat_window_tests.js

```diff
@@ -97,14 +97,24 @@ QUnit.test("Message post in chat window of chatter should log a note", async ()
     await contains(".o-mail-Message-content", {
         text: "A needaction message to have it in messaging menu",
     });
-    await contains(
-        ".o-mail-Message:contains(A needaction message to have it in messaging menu) .o-mail-Message-bubble.border"
-    ); // bordered bubble = "Send message" mode
+    await contains(".o-mail-Message", {
+        containsMulti: [
+            [
+                ".o-mail-Message-content",
+                { text: "A needaction message to have it in messaging menu" },
+            ],
+            [".o-mail-Message-bubble.border"], // bordered bubble = "Send message" mode
+        ],
+    });
```

Comment:
Again a bit sad it's less pretty and less readable, due to `text` being strict.
Would be nice in future to be more lax so assertion can be kept simple.